### PR TITLE
Merge "reportAnIssueMenuItem" and "submitFeedbackMenuItem"

### LINKED ISSUE #6180: Merge 'reportAnIssue' and 'submitFeedback'
https://github.com/brave/browser-laptop/pull/6080#issuecomment-266642253 Merge 'reportAnIssue' and 'submitFeedback'

- Brave Version: master branch

- Any related issues: #6179

### PR DESCRIPTION
**Pull request by bkilrain**
Deletes "Report an Issue" menu item because both it and "Submit Feedback"
link to community.brave.com.

Resolves #6180

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

## Test Plan:

Ensure `Report an Issue` no longer shows up in the `Help` menu

### CODE REVIEW

**Comment by bsclifton** [comment]
@bkilrain sorry for the huge delay here (finally checking this out now).  Will do an official review using GitHub and leave feedback :smile:

File: app/extensions/brave/locales/es/menu.properties [hunk 1]
```diff
@@ -87,7 +87,6 @@ preferences=Preferencias…
 settings=Ajustes…
 bookmarksManager=Administrador de marcadores…
 importBrowserData=Importar datos del navegador...
-reportAnIssue=Reportar un problema
```
Comment (bsclifton) [inline app/extensions/brave/locales/es/menu.properties]: The non-English (ex: `en-US`) files can be left alone; they'll get stripped out when the next internationalization pull happens :smile:

**Comment by bsclifton** [comment]
There were [3 failing tests on travis-ci](https://travis-ci.org/brave/browser-laptop/builds/188172709); I grabbed the code, rebased, and ran each test locally and confirmed they're working great. Merging :smile:

**Comment by bkilrain** [comment]
Yay! Thanks for all your help. I'll look for another issue~👌

**Pull request merged**